Orange/widgets/data/owtranspose.py

```diff
@@ -69,6 +95,11 @@ def __init__(self):
             "feature_names_column", contentsLength=12, searchable=True,
             callback=self._feature_combo_changed, model=self.feature_model)
 
+        self.remove_check = gui.checkBox(
+            gui.indentedBox(box, gui.checkButtonOffsetHint(button)), self,
+            "remove_redundant_inst", "Remove redundant instance",
+            callback=lambda: self.apply())
```

Comment:
Why not just `callback=self.apply`? Why a lambda function? (I am not criticizing, I genuinely want to know if there is something I do not understand.)